src/WaveTrack.cpp

```diff
@@ -420,6 +422,159 @@ const WaveClip* WaveTrack::FindClipByName(const wxString& name) const
    return nullptr;
 }
 
+void WaveTrack::PasteWaveTrack(double t0, const WaveTrack* other)
```

Comment:
Please insert this new member function immediately before WaveTrack::Paste().  That may make the commit more easily reviewable because there will be more common lines, and less in the diff view.